plugin/teksi_wastewater/interlis/gui/interlis_export_settings_dialog.ui

```diff
@@ -52,17 +52,35 @@
        <property name="checkable">
         <bool>true</bool>
        </property>
-       <layout class="QVBoxLayout" name="verticalLayout_2">
+       <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
-         <widget class="QLabel" name="labels_qgis_warning_label">
-          <property name="styleSheet">
-           <string notr="true">*{color: red; }</string>
-          </property>
+         <widget class="QLabel" name="orientation_label">
           <property name="text">
-           <string>This feature requires QGIS 3.26.2 or newer.</string>
+           <string>Angle for horizontal text in INTERLIS (default: 90.0°)</string>
```

Comment:
In other part of the dialog we use the term "label" and here "text". Should we choose one of them for consistency?
ìn INTERLIS` can be removed as already in the title of the dialog?

Reply:
OK adjusted